sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -59,7 +59,7 @@ export async function executeAtomXmlOperation(
     onUploadProgress: operationOptions.requestOptions?.onUploadProgress,
     onDownloadProgress: operationOptions.requestOptions?.onDownloadProgress,
     abortSignal: operationOptions.abortSignal,
-    spanOptions: operationOptions.tracingOptions?.spanOptions,
+    spanOptions: (operationOptions.tracingOptions as any)?.spanOptions,
```

Comment:
I think it would be kind if we added a comment for each one of these clients, so that developers of those clients can see why they have an ` as any` here :) otherwise this could be taken as some of that “black-magic that everybody does because it works” kinda thing